src/app/components/search/search-results/search-results.component.ts

```diff
@@ -87,7 +87,7 @@ export class SearchResultsComponent extends PageComponent implements OnInit {
         this.searchedWord = params.hasOwnProperty(this.queryParamName) ? params[this.queryParamName] : null;
         const query = this.formatSearchQuery(this.searchedWord, searchConfig['aca:fields']);
         if (query) {
-          this.queryBuilder.userQuery = decodeURIComponent(query);
+          this.queryBuilder.userQuery = decodeURIComponent(query).replace(/^https?:\/\//, '');
```

Comment:
this is not a valid solution in this way you are not going to search for what is required